spring-example/pom.xml

```diff
@@ -34,13 +34,13 @@
             <dependency>
```

Comment:
the graphiql starter dependency can be removed completely.  

beginning in v12.x of kickstart, all of the once separate tool starters were consolidated into `graphql-spring-boot- starter` and made property driven